permissions/policy1.json

```diff
@@ -101,7 +102,7 @@
         "dynamodb:UpdateItem"
       ],
       "Resource": [
-        "arn:aws:dynamodb:*:123456789012:table/atl_dc_*_tf_lock"
+        "arn:aws:dynamodb:*:123456789012:table/*"
```

Comment:
Probably better to have `"dynamodb:ListTables"` in a standalone statement if we're giving `*` resource?

Reply:
@yzha645 done